app/src/main/java/lodzka/politechnika/qrcode/model/User.java

```diff
@@ -0,0 +1,60 @@
+package lodzka.politechnika.qrcode.model;
+
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class User {
```

Comment:
Może warto zastosować Lomboka?

Reply:
ok myslalem nad tym to w nastepnych commitach juz zastosujemy lomboka